code/big_integer/src/big_integer.py

```diff
@@ -0,0 +1,44 @@
+class BigInteger(object):
+    BASE = 10
+    SIZE = 100
+
+    def __init__(self, digits=[]):
+        self.digits = [0 for i in range(self.SIZE)]
```

Comment:
What happens if we pass a string instead of an array ? Please add a check for incorrect data.